packaging/Dockerfile.centos_7_git_rpm

```diff
@@ -1,19 +1,9 @@
-# This Dockerfile is based on the recommendations provided in the
-# Centos official repository (https://hub.docker.com/_/centos/).
-# It enables systemd to be operational.
-FROM centos:7.5.1804
-ENV container docker
-COPY docker_enable_systemd.sh docker_sys_config.sh ./
-RUN bash ./docker_enable_systemd.sh && rm ./docker_enable_systemd.sh -f
+FROM nmstate/centos_7_base
```

Comment:
We are currently using nmstate/centos7-nmstate-* format, lets keep the convention.